src/__tests__/__snapshots__/module_identifiers.spec.js.snap

```diff
@@ -1,6 +1,18 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`should handle utility types 1`] = `
+exports[`should handle global types jsx 1`] = `
+"import * as React from \\"react\\";
+declare function s(node: React$Node): void;
+declare type Props = {
+  children: React$Node
+};
+declare class Component mixins React.Component<Props> {
```

Comment:
Why it uses `mixins` instead of `extends`?

Reply:
No idea, I didn't do anything to classes

Reply:
It also converts `implements` to `mixins`

Reply:
Hmm I wonder what is the purpose of it. There is no documentation for `mixins` for Flow.

Reply:
There was documentation on old website, it is basically `extends` without statics and multiple values through comma

Reply:
looks like there is no documentation because `mixins` is not supported: https://flow.org/try/#0PTAEAEDMBsHsHcBQBLAtgB1gJwC6gFSgCGAzqAEoCmRAxnpFrKqAORbV0sDciN0pZAMJNMAO0qi8qZAA9koslVo4AdMIyxxkgDwAFRuhIA+UAG8AvkA :D

Reply:
@villesau use `declare`

Reply:
hmm looks like it's supported for declarations: https://flow.org/try/#0PTAEAEDMBsHsHcBQBLAtgB1gJwC6gFSgCGAzqAEoCmRAxnpFrKqAORbV0sDciAJpTWhF2oQaTIBhJpgB2lGXlTIAHshlkqtHADopGWHIUAeAN4BfAHyhzQA

Reply:
So perhaps it's good enough.

Reply:
https://zhenyong.github.io/flowtype/docs/declarations.html#mixins